client/submitAssignment.js

```diff
@@ -120,7 +143,13 @@ function resetValidations() {
     formError.innerHTML = '';
 }
 
-// Verifies the file type (should be .zip, .tar or .tar.gz)
+/**
+ * Verifies the selected file input for valid file types and extensions.
+ * Allowed file types are 'application/zip', 'application/x-tar','application/gzip'.
+ * Displays error in 'formError' HTML element if a file is not selected within allowed types.
```

Comment:
we don't need to mention "'formError' HTML element"

Reply:
fix other occurrences of the same.

Reply:
Done